thunder/tests/distributed/test_ddp.py

```diff
@@ -1528,5 +1677,30 @@ def test_ddp_transformer_engine_llama_sanity(executor, devices, dtype):
     pass
 
 
+@instantiate(
+    dtypes=(thunder.float32,),
+    num_devices=2,
+    devicetypes=(devices.DeviceType.CUDA,),
+    executors=(TorchExecutor,),
+    decorators=(
+        # NOTE: ddp_wrapper
+        pytest.mark.parametrize(
+            "thunder_fsdp_strategy",
+            (
+                FSDPType.ZERO2,
+                FSDPType.ZERO3,
+            ),
+        ),
+        pytest.mark.skipif(not TE_AVAILABLE, reason="TransformerEngine is not installed."),
+        pytest.mark.skipif(not is_fp8_supported, reason=fp8_support_reason),
+        # NOTE: Setting `NVTE_TORCH_COMPILE`
```

Comment:
I think this note is not informative enough.
As I understand TE might use torch.compile internally and it doesn't work well with the multiprocessing setup in data parallel testing?

Reply:
Yes that is correct. This note should actually point to the note mentioned. Will update it to say `See NOTE: Setting `NVTE_TORCH_COMPILE`

https://github.com/Lightning-AI/lightning-thunder/blob/34d21e1455bf13e0c397207b018f2835acc3870a/thunder/tests/distributed/test_ddp.py#L1497-L1505